### PR TITLE
Implements a mechanical ERP system

### PR DESCRIPTION
**Pull request by LordFowl**
As per the discussion last year where we were interested in perhaps pursuing a laxening of the rules regarding ERP, I have noticed - with the privilege of my access to staff channels - little progress in actually implementing this initiative due to a virgin sense of inexperience among many of my fellow staff. Thus I believe that baby steps regarding the process are ideal, starting with a mechanically supported ERP system to acclimate staff and playerbase to the idea in a means we can control.

As such, I introduce ERP+:
- Two new mood meters have been added, lust and satisfaction. A high lust will distract you and slow you down, while a high satisfaction will distract you and speed you up. Lust increases over time and satisfaction decreases over time.
- Dragging yourself onto another person will open an ERP menu, provided there is mutual consent. There are three categories of ERP moves. Casual ERP will decrease lust and increase satisfaction. Intimate ERP will increase lust and satisfaction. Extreme ERP will increase satisfaction greatly and decrease lust greatly. Extreme ERP can only be done if you maintain your lust and satisfaction, while Intimate ERP is available provided you maintain your satisfaction. Casual ERP is always available. Too much ERP can be dangerous, however.
- There is a minor combo system that rewards a diverse moveset instead of just spamming the same moves. Also being naked improves gains.

- [x] Testing
- [x] Consider some possible sound effects.

### CODE REVIEW

**Comment by veraknocks** [comment]
'____'

**Comment by ComradeCorbyn** [comment]
honestly this is just epic

**Comment by Menown** [comment]
Oh boy, another mechanic I'll never use. Talk about bloat.

**Comment by Menown** [comment]
Wait, would this be considered bloat or inflation?

**Comment by Pacmandevil** [comment]
this is feature incomplete when compared to the rest of the game.
A good way to start fixing this would be adding brain traumas for foot fetishes, along with traumas for being attracted to Unathis, Tajara, and Varcua.

**Comment by Pacmandevil** [comment]
also i can do voice acting if u want hmu

File: code/modules/mob/living/carbon/human/human.dm [hunk 1]
```diff
@@ -1546,7 +1556,221 @@
...
+		dat += "<BR><A href='?src=\ref[src];erp=kiss_cheek'>Kiss [name]'s cheek</A>"
+		dat += "<BR><A href='?src=\ref[src];erp=cuddle'>Cuddle with [name]</A>"
+		dat += "<BR><A href='?src=\ref[src];erp=caress'>Caress [name] intimately</A>"
+		dat += "<BR><A href='?src=\ref[src];erp=sweet_nothings'>Whisper sweet nothings to [name]</A>"
+
+	//tier 3 ERPs. These decrease lust by 6 and increase satisfaction by 6 and prevent it from decreasing for a certain period, but are only available if lust is high enough.
+	if((Huser.satisfaction > 35) && (Huser.lust > 50))
+		dat += "<B><HR><FONT size=2>EXTREME ERP MOVES (DON'T TRY THIS AT HOME KIDS!)</FONT></B><BR><HR>"
+		dat += "<BR><A href='?src=\ref[src];erp=lewd_thoughts'>Share lewd thoughts with [name]</A>"
+		dat += "<BR><A href='?src=\ref[src];erp=touch_butt'>Touch butts with [name]</A>"
```
Comment (MarinaGryphon) [inline code/modules/mob/living/carbon/human/human.dm]: This is inappropriate. I have a suggestion: 
```suggestion
		dat += "<BR><A href='?src=\ref[src];erp=woohoo'>WooHoo with [name]</A>"
```

**Comment by alsoandanswer** [comment]
What the absolute fuck

File: code/modules/mob/living/carbon/human/human.dm [hunk 2]
```diff
@@ -1546,7 +1556,221 @@
...
+		dat += "<BR><A href='?src=\ref[src];erp=confess_admiration'>Confess romantic interest in [name]</A>"
+		dat += "<BR><A href='?src=\ref[src];erp=kiss_cheek'>Kiss [name]'s cheek</A>"
+		dat += "<BR><A href='?src=\ref[src];erp=cuddle'>Cuddle with [name]</A>"
+		dat += "<BR><A href='?src=\ref[src];erp=caress'>Caress [name] intimately</A>"
+		dat += "<BR><A href='?src=\ref[src];erp=sweet_nothings'>Whisper sweet nothings to [name]</A>"
+
+	//tier 3 ERPs. These decrease lust by 6 and increase satisfaction by 6 and prevent it from decreasing for a certain period, but are only available if lust is high enough.
+	if((Huser.satisfaction > 35) && (Huser.lust > 50))
+		dat += "<B><HR><FONT size=2>EXTREME ERP MOVES (DON'T TRY THIS AT HOME KIDS!)</FONT></B><BR><HR>"
+		dat += "<BR><A href='?src=\ref[src];erp=lewd_thoughts'>Share lewd thoughts with [name]</A>"
```
Comment (MarinaGryphon) [inline code/modules/mob/living/carbon/human/human.dm]: This should be restricted to Skrell only.

**Comment by LordFowl** [comment]
!review

**Pull request closed without merging**